example/spec/parallel/ingest/ingestFromPdrWithExecutionNamePrefixSpec.js

```diff
@@ -193,10 +193,11 @@ describe('The DiscoverAndQueuePdrsExecutionPrefix workflow', () => {
       granuleId: testDataGranuleId,
     });
     // clean up stack state added by test
-    await apiTestUtils.deletePdr({
-      prefix: config.stackName,
-      pdr: pdrFilename,
-    });
+    await waitAndDeletePdr(
```

Comment:
So interesting anecdote from another pairing session  - the call to `waitForCompletedExecution` is waiting for a *step function* call, not an API call.     This will certainly fix an issue with the PDR not having been removed, but perhaps a wait on the execution status would resolve it plus others?

Reply:
I'm fine with merging this, but if not added here will probably update the above call.

Reply:
https://github.com/nasa/cumulus/pull/2372/commits/6f39390997f3dfeafe6502e46850dce10fb7d5d7